example-projects/game-processor/scenario.yaml

```diff
@@ -1,11 +1,42 @@
-version: 0.0.1
-plot-1:
-  eps: 1
-  priority: 2
-  payload:
-    test:
-      subtest: '2'
-  parameters:
-    serverId: '1'
-    playerId: '5'
-    itemId: '3'
+version: "0.0.1"
+user-logs-on:
+  game/server/{serverId}/events/player/{playerId}/connect:
+    playerId :
+      min: 0
+      max: 2000
+    serverId:
+      min: 0
+      max: 4
+user-gameLoop:
+  loop:
+    interval:
+      600
+    cycles:
+      5
```

Comment:
Feel free to create multiple scenarios, so you don't change existing useable scenarios 🙂

Reply:
i will try to do it in a future pull request